hack/integration-test/wait-for-ready.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+condition="Ready"
+timeout="10m"
+namespace="image-based-install-operator"
+selector="app=image-based-install-operator"
+
+echo "Waiting for (pod) on namespace (${namespace}) with labels (${selector}) to be created..."
```

Comment:
nit: isn't it "in namespace", i.e. not on :)

Reply:
Yeah I can change this wording. I actually copied a bunch of this from assisted so I probably didn't review it closely enough :sweat_smile: